### ISSUE TITLE
Wrong price with table rates for Belgium/Netherlands

### ISSUE THREAD

**Issue by DeLuuk** [myparcelnl/magento#499]
**Describe the bug**

**To Reproduce**
1. Configure the MyParcel shipment to use Table Rates as method.
2. Have a different price for NL and BE like this:
```
Country,Region/State,"Zip/Postal Code","Weight (and above)","Shipping Price"
NLD,*,*,0.0000,7.0000
NLD,*,*,30.0000,49.0000
BEL,*,*,0.0000,9.9500
BEL,*,*,23.0000,49.0000
```
3. On the front-end, default checkout, fill in a Dutch address
4. Change it to a Belgium address

**Expected behavior**
The price should change from €7 to €9.95 but it stays at €7.
Thing is, it does change it but not visually. If you press next while the price says €7 with a Belgium address the price will be €9.95 on the next step.

**Screenshots**
Netherlands:
![Screenshot 2020-10-15 at 13 43 18](https://user-images.githubusercontent.com/26225521/96118992-9b916580-0eec-11eb-97ab-2e43762b90f3.png)
Belgium:
![Screenshot 2020-10-15 at 13 42 56](https://user-images.githubusercontent.com/26225521/96118978-96ccb180-0eec-11eb-8799-0cc8fa39921b.png)

**Desktop (please complete the following information):**
 - Magento 2.3.4
 - MyParcel Plugin version 4.1.2
 - MyParcel SDK version 5.2.1

**Comment by Mitchell-MyParcel** [comment]
@DeLuuk Could you try updating to 4.1.3. to check if the issue persists?

**Comment by Kroellie** [comment]
Probably same issue as the bug posted in #530: the rates are not refreshed when parameters change... it would help if we could determine the optimal refresh routine as suggested by me in that ticket.

**Comment by RichardPerdaan** [comment]
Solved in [4.1.6-beta.1](https://github.com/myparcelnl/magento/releases/tag/4.1.6-beta.1)
